packages/web-components/src/theme/set-theme.stories.ts

```diff
@@ -2,15 +2,92 @@ import { html } from '@microsoft/fast-element';
 import { teamsDarkTheme, teamsLightTheme, webDarkTheme, webLightTheme } from '@fluentui/tokens';
 
 import { renderComponent } from '../helpers.stories.js';
-import { setTheme } from './set-theme.js';
+import { setTheme, type Theme } from './set-theme.js';
```

Comment:
I remember there being issues with a few tools when using combined imports. It's probably best to keep these separated:

```suggestion
import type { Theme } from './set-theme.js';
import { setTheme } from './set-theme.js';
```